controllers/controllers.js

```diff
@@ -1,11 +1,17 @@
-const {selectTopics} = require('../models/models')
+const { selectTopics, selectEndpoints } = require('../models/models')
 
-exports.getTopics = (req,res,next) => { 
+exports.getTopics = (req, res, next) => {
     selectTopics().then((data) => {
-        res.status(200).send({data})
-    
-    })
-    .catch(err => {
-        next(err)
+        res.status(200).send({ data })
+
     })
+        .catch(err => {
+            next(err)
+        })
+};
+
+exports.getEndpoints = (req, res, next) => {
+    const endpoints = selectEndpoints();
+    res.status(200).send(endpoints)
+    
```

Comment:
Nice, but we dont actually need next here so it doent need to be included in the params.